singer_sdk/configuration/_dict_config.py

```diff
@@ -99,3 +99,18 @@ def merge_config_sources(
         config.update(read_json_file(config_path))
 
     return config
+
+
+def merge_missing_config_jsonschema(
```

Comment:
Refactored to a helper function in this private module. Previously it was a local function in the base class's method.